setup/native-script.rb

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+
+# Only the user can manually download and install Xcode from AppStore
+puts "Installing Xcode... Please, click 'Get' or 'Update' to install Xcode from the App Store."
```

Comment:
We are opening the AppStore to let the user install Xcode. We are not launching the Xcode installer, per se. The message was suggested by @ikoevska and I think it is appropriate in our case.